scripts/inner/kubestellar-kube-bind

```diff
@@ -56,15 +71,28 @@ then
     exit 1
 fi
 
-conf_file="/tmp/$space_name.conf"
-kubectl-kubestellar-get-config-for-space --space-name $space_name --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT --in-cluster --output $conf_file
-KUBECONFIG=$conf_file
+# Some place protectable to store kubeconfig files.
+# We do not have promises about what the $PWD is.
+# Some users administer multiple KubeStellar systems,
+# so the name must not be a pure function of the space name.
```

Comment:
This is the first time I see requirement of running multiple KS instances in parallel, I believe this is not supported by bare-processes, and I'm not sure it was ever tested for QS tests (that use  some local files as well ...)

Reply:
I _know_ that @clubanderson is managing multiple MCAD/kubeflow stacks.